modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_buttons.scss

```diff
@@ -17,4 +17,14 @@
     @include modular-spacing('margin-bottom', 0);
   }
 
+  // If they're in the same paragraph, get them to stack.
+  [class^="su-button"] {
```

Comment:
this says the classes must start with `su-button`. so then an element like `<a class="foo-bar su-button">` would not be targeted. I think you would rather have the selector `[class*="su-button"]`

Reply:
Why not just `.su-button`? 

I agree with @pookmish in that the specificity of the order of the classes cannot be guaranteed and we should write css to support the classes that show up in any order.

Reply:
```suggestion
  .su-button {
```